.github/workflows/ci_rust.yml

```diff
@@ -168,7 +205,7 @@ jobs:
       # We don't need to format the generated files,
       # but if they don't exist other code breaks.
       - name: Generate
-        run: ./${{env.ROOT_PATH}}/generate.sh
+        run: ./${{env.ROOT_PATH}}/generate.sh --skip-tests
 
```

Comment:
Why are we skipping tests?

Reply:
This is a `fmt` task. Tests are run as part of the other tasks above.